opensfm/commands/undistort.py

```diff
@@ -121,13 +130,25 @@ def undistort_image(arguments):
             undistorted = render_perspective_view_of_a_panorama(
                 image, shot, subshot,
                 cv2.INTER_LINEAR if interpolation == cv2.INTER_AREA else interpolation)
-            getattr(data, save_image)(subshot.id, undistorted)
+            undistorted = scale_image(undistorted, image_scale)
+            getattr(data, save_image)(subshot.id, undistorted, image_format)
     else:
         raise NotImplementedError(
             'Undistort not implemented for projection type: {}'.format(
                 shot.camera.projection_type))
 
 
+def scale_image(image, scale_factor):
+    """Scale an image by a factor."""
+    if scale_factor == 1.0:
+        return image
+    
+    height, width, _ = image.shape
+    width = int(width * scale_factor)
+    height = int(height * scale_factor)
+    return cv2.resize(image, (width, height), interpolation=cv2.INTER_NEAREST)
```

Comment:
For downsampling, we should use `cv2.INTER_AREA` to reduce aliasing artifacts.

Reply:
For the purpose of texturing meshes, in my tests I've tried both. Using cv2.INTER_AREA I noticed that the images are obviously smoother, but adds significant blur in the final textured mesh. I guess the best choice would depend on the use case. I could add an extra flag to specify what interpolation method to use?

Reply:
No need for an option there. It would add complexity and we can pass without it.